vendor/bat-native-ads/src/bat/ads/internal/deprecated/client/client_info.cc

```diff
@@ -64,8 +64,8 @@ base::Value::Dict ClientInfo::ToValue() const {
   base::Value::Dict advertisers;
   for (const auto& [key, value] : seen_advertisers) {
     base::Value::Dict advertiser;
-    for (const auto& [key, value] : value) {
-      advertiser.Set(key, value);
+    for (const auto& [ad_key, ad_value] : value) {
```

Comment:
do we need the verbosity in this file for key/value? (There is another open issue to cleanup structured bindings.

Reply:
Ah yes they are shadowed.